timesketch/lib/analyzers/contrib/hashlookup_analyzer.py

```diff
@@ -0,0 +1,115 @@
+"""Index analyzer plugin for Hashlookup."""
+
+import logging
+
+from flask import current_app
+import requests
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+from timesketch.lib import emojis
+
+
+logger = logging.getLogger("timesketch.analyzers.hashlookup")
+
+
+class HashlookupAnalyzer(interface.BaseAnalyzer):
+    """Analyzer for Hashlookup."""
+
+    NAME = "hashlookup_analyzer"
+    DISPLAY_NAME = "Hashlookup"
+    DESCRIPTION = "Mark events using Hashlookup"
+
+    def __init__(self, index_name, sketch_id, timeline_id=None, **kwargs):
+        """Initialize the Analyzer.
+
+        Args:
+            index_name: OpenSearch index name
+            sketch_id: The ID of the sketch.
+            timeline_id: The ID of the timeline.
+        """
+        super().__init__(index_name, sketch_id, timeline_id=timeline_id)
+        self.hashlookup_url = kwargs.get("hashlookup_url")
+        self.cp = 0
```

Comment:
What's cp here? I gather it's some sort of counter for succesful lookups, but can we make the attribute name more explicit?

Reply:
FIxed in 71f1bffe00ecc857c5ad1e9af69818badb06491c